docs/proximity-hcsr04-i2c.md

```diff
@@ -0,0 +1,67 @@
+<!--remove-start-->
+
+# Proximity - HC-SR04
+
+<!--remove-end-->
+
+
+Ping proximity example for use with an `HCSR04` sensor and I2C backpack. Standard
+firmata should be used or any other standard I2C capable IO Plugin.
```

Comment:
Should we add a note here and a link to https://github.com/ajfisher/nodebots-hcsr04?